index.ts

```diff
@@ -14,7 +14,13 @@ const templates = {
 
 catalog.heuristics.map(h => {
     h.body = markdown.toHTML(fs.readFileSync(h.filename, 'utf8'));
-
+    h.terms = h.terms.map(t => {
+        const key = t;
+        return {
+            title:markdown.toHTML(t).slice(3, -4),
+            key: t
+        }
+    });
```

Comment:
Apparently there's a merge conflict, possibly with an old pull-request. I'll use this code since all the relevant pull requests came from you and this is your most recent code.